core/src/main/java/com/web3auth/core/types/MfaSetting.kt

```diff
@@ -0,0 +1,7 @@
+package com.web3auth.core.types
+
+data class MfaSetting(
+    var enable: Boolean?,
```

Comment:
enable is mandatory, priority is optional, mandatory is optional

Reply:
Resolved